checks/config.go

```diff
@@ -57,12 +59,24 @@ type Config struct {
 	// []string{".*", "_*"}.  This is a glob that is applied to each path
 	// component of each file.
 	IgnorePatterns []string `yaml:"ignore_patterns"`
+
+	// MaxConcurrent, if not zero, is the maximum number of concurrent processes
```

Comment:
I think you are right and it's probably only appropriate on the command line and not in the committed file. As such, this could be removed.

Sorry for taking so long to reply; I had an issue with git 2.6 that I wanted to clear out first and I got a bit overwhelmed.

You'll have to merge/rebase first.